content/blog/2020-09-09-september-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,188 @@
+---
+title: September '20 Heartbeat
+date: 2020-09-09
+description: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+descriptionLong: |
+  This month, catch us on the Software Engineering Daily Podcast, 
+  check out our favorite DVC and CML tutorials and projects, and 
+  celebrate 1000 YouTube subscribers!
+picture: 2020-09-09/header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/september-20-heartbeat/488
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - R
+  - Meetup
+  - Videos
+---
+
+Welcome to our September roundup of cool news, new releases, and recommended
+reading!
+
+## News
+
+### Dmitry on Software Engineering Daily
+
+Our CEO Dmitry Petrov was interviewed on the much-beloved Software Engineering
+Daily podcast! Host [Jeff Meyerson](https://twitter.com/the_prion) kicked off
+the discussion:
+
+> Code is version controlled through git, the version control system originally
+> built to manage the Linux codebase. For decades, software has been developed
+> using git for version control. More recently, data engineering has become an
+> unavoidable facet of software development. It is reasonable to ask–why are we
+> not version controlling our data?
+
+For the rest of the episode, listen here!
+
+<external-link
+href="https://softwareengineeringdaily.com/2020/08/24/data-version-control-with-dmitry-petrov/"
+title="Data Version Control with Dmitry Petrov"
+description=""
+link="softwareengineeringdaily.com"
+image="/uploads/images/2020-09-09/sedaily.jpeg"/>
+
+### Contributor's meetup
+
+Last week, we held a meetup for contributors to DVC! Core maintainer
+[Ruslan Kupriev](https://github.com/efiop) hosted a get-together for folks who
+contribute new features, bug fixes, and more to the community. If you missed it,
+you can watch it on YouTube.
+
+https://youtu.be/jUYSTERXxWg
+
+### New videos
+
+We've released several new videos to our growing
```

Comment:
You can emphasize even more on the success - like "we recently started blogging and have already reached 1000 subscribers" or more personal "I remember our first video tutorial - it was a month ago but I remember it like yesterday". 

You can also mention how great this youtube community is - comment, same users show up over and over again in the comments.

Reply:
yep, i expanded it!